native/effektio/Cargo.toml

```diff
@@ -77,9 +80,9 @@ android_logger = "0.12"
 tokio = { version = "1", features = ["rt-multi-thread", "parking_lot"] }
 tracing-android = "0.2"
 
-#   ----   DESKTOP
-[target.'cfg(all(not(target_os = "android"), not(target_arch = "wasm32"), not(target_os = "ios")))'.dependencies]
-flexi_logger = "0.24.0"
```

Comment:
Why did you remove flexilogger for desktop setups? The setup below is only tied to macos desktops. What about Linux and Windows desktops?

Reply:
I confirmed `fern` instead of `flexi_logger` is working on windows and macos.
It is needed to reduce unnecessary libraries, because of ruby-macho in ios build.